src/components/DesktopContainer.jsx

```diff
@@ -16,6 +16,36 @@ import { HomepageHeading } from "./HomepageHeading";
  */
 class DesktopContainer extends Component {
   state = {};
+  
+  adminWebsite = () =>{
+    window.open(
+        process.env.REACT_APP_ADMIN_SITE_URL
+      )
+  };
+  scrollToTop = () =>{
+    window.scrollTo({
+      top: 0, 
+      behavior: 'smooth'
+    });
+  };
+  scrollToSpaces = () =>{
+    window.scrollTo({
+      top: 700, 
```

Comment:
These constant values can change depending on the device height/width. One option is to use anchor tag (ie: `<a href="#spaces" />`) and another option would be to do it programmatically with https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

Reply:
Also, check and see if the `react-router-dom` docs have any support for scrolling to anchor tags within a page.